pkg/frontend/admin_openshiftcluster_redeployvm.go

```diff
@@ -24,9 +24,12 @@ func (f *frontend) postAdminOpenShiftClusterRedeployVM(w http.ResponseWriter, r
 }
 
 func (f *frontend) _postAdminOpenShiftClusterRedeployVM(log *logrus.Entry, ctx context.Context, r *http.Request) error {
-	vars := mux.Vars(r)
 	vmName := r.URL.Query().Get("vmName")
-	action, _, err := f.prepareAdminActions(log, ctx, vmName, strings.TrimPrefix(r.URL.Path, "/admin"), vars)
+	resourceName := chi.URLParam(r, "resourceName")
+	resourceType := chi.URLParam(r, "resourceType")
+	resourceGroupName := chi.URLParam(r, "resourceGroupName")
```

Comment:
what is the difference between this way of getting URL params and another way we use e.g. in pkg/frontend/admin_openshiftcluster_reconcilefailednic.go

Reply:
none, I just replaced in place the existing variables

Reply:
does it make sense then to have one style of getting URL params?

Reply:
golang allows both and there isn't any logic difference, I don't really have preferences towards it, but maybe you can bring it up with the wider team

Reply:
And we can always change it during a cleanup.